src/main/java/org/folio/rest/impl/ClaimingApi.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.jaxrs.model.ClaimingCollection;
+import org.folio.rest.jaxrs.resource.OrdersClaim;
+import org.folio.service.claiming.ClaimingService;
+import org.folio.spring.SpringContextUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.ws.rs.core.Response;
+import java.util.Map;
+
+import static org.folio.orders.utils.ResourcePathResolver.CLAIMING_BUSINESS;
+import static org.folio.orders.utils.ResourcePathResolver.resourceByIdPath;
+import static org.folio.rest.RestConstants.OKAPI_URL;
+
+public class ClaimingApi extends BaseApi implements OrdersClaim {
```

Comment:
should be renames to PiecesCLaimingApi and also all corresponding places, documentation etc